ps_faviconnotificationbo.php

```diff
@@ -0,0 +1,296 @@
+* If you did not receive a copy of the license and are unable to
+* obtain it through the world-wide-web, please send an email
+* to license@prestashop.com so we can send you a copy immediately.
+*
+* DISCLAIMER
+*
+* Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+* versions in the future. If you wish to customize PrestaShop for your
+* needs please refer to http://www.prestashop.com for more information.
+*
+*  @author    PrestaShop SA <contact@prestashop.com>
+*  @copyright 2007-2018 PrestaShop SA
+*  @license   http://opensource.org/licenses/afl-3.0.php  Academic Free License (AFL 3.0)
+*  International Registered Trademark & Property of PrestaShop SA
+*/
+
+if (!defined('_PS_VERSION_')) {
+    exit;
+}
+
+class Ps_faviconnotificationbo extends Module
+{
+    protected static $conf_fields = array(
+    'BACKGROUND_COLOR_FAVICONBO',
+    'TEXT_COLOR_FAVICONBO',
+    'CHECKBOX_ORDER',
+    'CHECKBOX_CUSTOMER',
+    'CHECKBOX_MESSAGE'
+    );
+
+    public function __construct()
+    {
+        $this->name = 'ps_faviconnotificationbo';
+        $this->tab = 'administration';
+        $this->version = '1.0.0';
+        $this->author = 'PrestaShop';
+        $this->bootstrap = true;
+
+        $this->module_key = '91315ca88851b6c2852ee4be0c59b7b1';
+        $this->author_address = '0x64aa3c1e4034d07015f639b0e171b0d7b27d01aa';
+
+        $this->controllers = array(
+            'adminAjax' => 'AdminAjaxFaviconBO',
+        );
+
+        parent::__construct();
+
+        $this->displayName = $this->l('Order Notifications on the Favicon');
+        $this->description = $this->l('Get notified when you get new order, new client or new message directly on your browser tab of your back office even if your working on another page');
+
+        // Settings paths
+        $this->js_path = $this->_path.'views/js/';
+        $this->css_path = $this->_path.'views/css/';
+        $this->img_path = $this->_path.'views/img/';
+        $this->docs_path = $this->_path.'docs/';
+        $this->logo_path = $this->_path.'logo.png';
+        $this->module_path = $this->_path;
+        $this->ps_version = (bool)version_compare(_PS_VERSION_, '1.7', '>=');
+
+        // Confirm uninstall
+        $this->confirmUninstall = $this->l('Are you sure you want to uninstall this module?');
+        $this->ps_versions_compliancy = array('min' => '1.5', 'max' => _PS_VERSION_);
+    }
+
+    /**
+     *
+     *
+     * @param none
+     * @return bool
+     */
+    public function install()
+    {
+        Configuration::updateValue('CHECKBOX_ORDER', '1');
+        Configuration::updateValue('CHECKBOX_CUSTOMER', '1');
+        Configuration::updateValue('CHECKBOX_MESSAGE', '1');
+        Configuration::updateValue('BACKGROUND_COLOR_FAVICONBO', '#DF0067');
+        Configuration::updateValue('TEXT_COLOR_FAVICONBO', '#ffffff');
+        if (parent::install() && $this->registerHook('BackOfficeHeader') && $this->installTab() == false) {
```

Comment:
Please double check this line, which seems to return false when everything goes fine.

We could expect 

```php
return parent::install() && $this->registerHook('BackOfficeHeader') && $this->installTab();
```